phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -973,6 +976,25 @@ private PTable getTable(byte[] tenantId, byte[] schemaName, byte[] tableName, lo
         return table;
     }
 
+    private PName getPhysicalTableName(Region region, byte[] tenantId, byte[] schema, byte[] table, long timestamp) throws IOException {
```

Comment:
You think adding this to SchemaUtil make any sense. I see a lot of implementation of getPhysicalTableName